docs/source/src_code/model_components/networks.rst

```diff
@@ -6,7 +6,25 @@ Networks
 The network class is a subclass of the ModelComponent class. All networks are modelled using this class, and there are
 no subclasses of specific network models. An overview of all networks that are currently modelled can be found
 :ref:`below <network_list>`. A network is defined as the set of all arcs (i.e., connections between nodes) of a
-specific network type (e.g., "electricitySimple").
+specific network type (e.g., "electricitySimple"). In addition to the performance and
+cost parameters in defined in the respective json file of the network, networks can
+generally be modelled as either bi- or uni-directional by modifying the parameter
+``allow_only_one_direction`` and ``allow_only_one_direction_precise`` in the
+json file of the respective technology.
+
+If ``allow_only_one_direction = 1`` the following properties of the network are
+enforced:
+
+- The size of an arc in both direction needs to be equal.
```

Comment:
If you allow only one direction (ie through a pipeline), you could have different sizes for both directions, right? Only in case of bi directional the size should be the same cause you have only one pipeline segment